docs/package_basics.md

```diff
@@ -0,0 +1,15 @@
+# Package Basics
+DC/OS uses a custom packaging tool called [pkgpanda](../pkgpanda) which operates on a "package tree". The [packages](../packages) directory is the root of all the packages and `pkgpanda` is built to construct it into a "package tree". A "package tree" is constructed by looking in each directory in a root and parsing a `buildinfo.json` and `build` file from each directory. If these are present, then the directory is considered a package and added to the tree. This tool can read these metadata JSONs in a "package tree", verify the package dependency list is resolvable by walking through dependent packages, and then use the build script files accompanying the metadata to perform a Docker-based build. Finally, these artifacts are packaged in XZ-compressed tar-balls. For more information on why these design choices were made, take a look at the [docs](https://dcos.io/docs/1.9/overview/design/installation/).
```

Comment:
Mentioning Docker and XZ feels overly detailed for a basics doc. Probably better covered in a more detailed doc under `pkgpanda/`.